web/src/main/java/org/akaza/openclinica/service/ImportServiceImpl.java

```diff
@@ -125,7 +125,7 @@
     SimpleDateFormat sdf_logFile = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'");
```

Comment:
Just pointing it out. Not needed
Unused imports
Add Access restrictions to all autowired fields

Reply:
Done